packages/twenty-front/src/modules/ui/layout/tab/states/activeTabIdComponentState.ts

```diff
@@ -1,6 +1,8 @@
-import { createComponentState } from '@/ui/utilities/state/component-state/utils/createComponentState';
+import { ActiveTabComponentInstanceContext } from '@/ui/layout/tab/states/contexts/ActiveTabComponentInstanceContext';
+import { createComponentStateV2 } from '@/ui/utilities/state/component-state/utils/createComponentStateV2';
 
-export const activeTabIdComponentState = createComponentState<string | null>({
+export const activeTabIdComponentState = createComponentStateV2<string | null>({
```

Comment:
shouldn't this be part of the TabList component?
Shouldn't the componentInstanceContext be TabListComponentInstanceContext?